src/lib.rs

```diff
@@ -38,6 +37,43 @@ fn get_args(current_dir: &OsStr) -> clap::ArgMatches {
         .get_matches()
 }
 
+fn run_checks(fe: &FileEntry) -> io::Result<Vec<Warning>> {
+    let mut f = File::open(fe.path.clone())?;
```

Comment:
Is calling the `clone()` method necessary?

Reply:
No, I have used ref.